### ISSUE TITLE
Is this protocol for the old API? 

### ISSUE THREAD

**Issue by MarcAmil30** [abiedermann/openblend_public#1]
When running this protocol. I got this error :

```
The python protocol you uploaded has the Python API Version 1.0.  Robot server version 4.0.0 is
the official end of life of Python API Version 1.0. The minimum supported Python API Version is 2.0. This means that this protocol
will not run in robot server version 4.0.0 and above.
Please downgrade your robot server version if you wish to run this protocol. Otherwise, please upgrade this
protocol to Python API Version 2.0 or above.
```

I saw that it got updated recently, therefore I wanted to ask if this repo was modified for a new API?

**Comment by abiedermann** [comment]
Hey Marc, sorry you've run into this issue. We haven't planned to update the API. For us, it's been simpler to revert the app back to Opentrons software version 3.16.1 we developed under rather than refactor the source code.

**Comment by MarcAmil30** [comment]
I am new to the opentron, would this software version (3.16.1) still work for the OT-2? Additionally, is there a way to simulate this code without getting my hands on the robot?

**Comment by abiedermann** [comment]
Yes, it should work with the 3.16.1 software version. It's probably not compatible with simulation. Basically, the command line interface is a workaround that allows you to work with csv file inputs (idk if this has been implemented by Opentrons in later updates, but it wasn't at the time I wrote the code). You could run a simple program (see example_templates folder) without setting up the deck to get a feel for it. Entering Control+C into the command line would let you stop the run early if anything strange happened.
